CONTRIBUTORS-GUIDE.md

```diff
@@ -20,7 +20,7 @@ Split Redux SDK is an open source project and we welcome feedback and contributi
 
 ### Building the SDK
 
-For the moment, we provide two different builds of the library: 
+For the moment, we provide two different builds of the library:
```

Comment:
Better use `At the moment`, rather than `For the moment`

Reply:
Done! Thanks Emi